docs_src/design/adr/device-service/0012-DeviceService-Filters.md

```diff
@@ -0,0 +1,152 @@
+# Device Service Filters
+
+## Status
+
+### Proposed
+
+- design for Hanoi
```

Comment:
Hanoi's already out the door, so I think this should be updated to design for Ireland.

Reply:
addressed